src/main/resources/project/project.xml

```diff
@@ -1420,7 +1436,14 @@
                 <formalParameterName>availability_zone</formalParameterName>
                 <defaultValue>1</defaultValue>
                 <description/>
-                <required>1</required>
+                <required>0</required>
+                <type>entry</type>
+            </formalParameter>
+            <formalParameter>
+                <formalParameterName>region</formalParameterName>
+                <defaultValue>1</defaultValue>
```

Comment:
Because availability_zone also has default value = 1. And all other optional fields too.